tests/conftest.py

```diff
@@ -31,7 +31,7 @@ def client(app):
 @pytest.fixture
 def one_task(app):
     new_task = Task(
-        title="Go on my daily walk 🏞", description="Notice something new every day", completed_at=None)
+        title="Go on my daily walk 🏞",description="Notice something new every day",completed_at=None)
```

Comment:
I would put the space back. It's affecting a few of your tests passing. 

```suggestion
        title="Go on my daily walk 🏞", description="Notice something new every day", completed_at=None)
```